openmls/src/group/core_group/staged_commit.rs

```diff
@@ -138,9 +137,11 @@ impl CoreGroup {
 
         let ciphersuite = self.ciphersuite();
 
-        let (commit, proposal_queue, sender_index) =
-            self.public_group
-                .validate_commit(mls_content, proposal_store, provider.crypto())?;
+        let (commit, proposal_queue, sender_index) = self.public_group.validate_commit(
+            mls_content,
+            self.proposal_store(),
```

Comment:
Wouldn't it make more sense if `PublicGroup::validation_commit` used `self.proposal_store` here? Because right now the CoreGroup first fetches the value from the PublicGroup and then passes it back into the method of PublicGroup.